.github/workflows/pytest.yml

```diff
@@ -43,7 +49,24 @@ jobs:
       - name: "Run pytest"
         env:
           SUBMAN_TEST_IN_CONTAINER: "1"
-          PYTEST_ADDOPTS: "--color=yes --code-highlight=yes --showlocals"
+          PYTEST_ADDOPTS:
+            "--color=yes --code-highlight=yes --showlocals
+            --cov 'src/' --cov-report 'term:skip-covered'
+            --cov-report 'xml:/tmp/coverage.xml' --junitxml '/tmp/pytest.xml'"
         run: |
           dbus-run-session \
             python3 -m pytest ${{ matrix.pytest_args }}
+
+      - name: "Publish coverage"
+        uses: MishaKav/pytest-coverage-comment@main
+        if:
+          ${{
+            github.event.pull_request.head.repo.full_name == github.repository
+            &&
+            matrix.name == 'Fedora latest'
+          }}
```

Comment:
hmm this won't work in yaml, as it is a proper multiline block; also i think the `${{...}}` is not needed here -- try something like:
```yaml
  if: |
    github.event.pull_request.head.repo.full_name == github.repository
    &&
    matrix.name == 'Fedora latest'
```
references:
- https://github.com/orgs/community/discussions/25641#discussioncomment-3248571
- https://yaml.org/spec/1.2.2/#chapter-8-block-style-productions